client/src/App.js

```diff
@@ -100,6 +100,11 @@ const App = () => {
             minRole="admin"
           />
 
+          <Route
+            path="/saved"
+            render={() => <Resources authed={authed} saved />}
```

Comment:
Could we change the render line to `render={props => <Resources {...props} authed={authed} saved />}`?  I think I might've said to ignore the props, but it seems it causes a bug with filtering since we can't push to the router's history anymore.